FuncGodot Manual/pages/tips_runtime_building.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+    <title>FuncGodot Manual</title>
+    <link rel="icon" type="image/x-icon" href="../images/godot_ranger.svg">
+    <link rel="stylesheet" href="../styles.css">
+</head>
+<body>
+    <div class="main">
+        <h1>Runtime map building</h1>
+        <p>Runtime map building is where you build a map in the middle of a running game. This can be used for a number of purposes such as: procedural generation of maps, loading maps from the .map file instead of scenes, and to support user-made custom maps. Runtime map building can also be combined with map editor launch configurations to provide a seamless experience making a map and seeing it in-game.</p>
+        <h2>Setting up Runtime map building in Godot</h2>
+        <p>Runtime map building involves using the method <code>verify_and_build()</code> available under the <a href="ref_func_godot_map.html" target="main">FuncGodotMap</a> node. The idea behind this, is that when you build a map at runtime, you set the <code>map_file</code> field, and then call this method. When this is done, this builds all the entities in the normal way, calling <code>_func_godot_apply_properties(entity_properties: Dictionary)</code> on these entities.</p>
```

Comment:
Reworded to respect operation order and sharpened focus. Also adds an extra exports consideration note.

<p>To build a map at Runtime, start by either using a templated scene with an empty <i>FuncGodotMap</i> or instantiating one. Next you'll need to make sure to set either the <code>local_map_file</code> or <code>global_map_file</code> to your map's file path. Don't use <code>load()</code> here! That will be taken care of in the next step.</p>

<blockquote><i>NOTE: After exporting your project, packed map resources may end up having their extensions changed by Godot! If you're having problems with loading maps in an exported project, try checking if your map files have been renamed using a PCK explorer.</i></blockquote>

<p>Once your map file path has been set, you can run the <code>verify_and_build()</code> method from the <a href="ref_func_godot_map.html" target="main">FuncGodotMap</a> node. When this is done, your map will be built just as it would have been by pressing the <i>Build</i> button in the Editor, along with all of the entity callbacks. Be aware that the <code>Engine.is_editor_hint()</code> conditional will return as false, in case you use it to control properties handling through the FuncGodot entity callbacks.</p>